content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -63,6 +63,24 @@ resources:
           keys:
             - name: key1
               secret: YWJjZGVmZ2hpamtsbW5vcHFyc3R1dnd4eXoxMjM0NTY=
+  - resources:
+      - events
+    providers:
+      - identity: {} # do not encrypt events even though  *.* is specified below
+  - resources:
+      - '*.apps'
```

Comment:
We probably want to mention the minimum supported version `v1.27` for using wildcards? Similar to "encrypt custom resource" here https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/#before-you-begin